mne/report.py

```diff
@@ -769,6 +769,9 @@ def _add_figs_to_section(self, figs, captions, section='custom',
             warnings.warn('Could not import mayavi. Trying to render '
                           '`mayavi.core.scene.Scene` figure instances'
                           ' will throw an error.')
+        if np.isscalar(scale) and scale < 1.:
+            raise ValueError('scale, if float, must be >= 1 not %s'
+                             % scale)
```

Comment:
I would just leave it as is and add a comment to the docs.

2015-04-29 19:50 GMT+02:00 Denis-Alexander Engemann <
denis.engemann@gmail.com>:

> But there are contexts in whichs scales < 1 can work.
>
> 2015-04-29 19:49 GMT+02:00 Eric Larson <notifications@github.com>:
>
>> In mne/report.py
>> <https://github.com/mne-tools/mne-python/pull/2039#discussion_r29361265>:
>>
>> > @@ -769,6 +769,9 @@ def _add_figs_to_section(self, figs, captions, section='custom',
>> >              warnings.warn('Could not import mayavi. Trying to render '
>> >                            '`mayavi.core.scene.Scene` figure instances'
>> >                            ' will throw an error.')
>> > +        if np.isscalar(scale) and scale < 1.:
>> > +            raise ValueError('scale, if float, must be >= 1 not %s'
>> > +                             % scale)
>>
>> The discussion here indicates that scale >= 1 is the only safe condition:
>>
>> #2014 (comment)
>> <https://github.com/mne-tools/mne-python/issues/2014#issue-70542876>
>>
>> If you have a better solution I'm all ears...
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mne-tools/mne-python/pull/2039/files#r29361265>.
>>
>
>